system-tests/projects/sourcemaps/cypress-disabled.config.mjs

```diff
@@ -0,0 +1,16 @@
+import path from 'node:path'
+import os from 'node:os'
+import cypressPreprocessor from './cypress-preprocessor-disabled.config.mjs'
+
+export default {
```

Comment:
do we need to wrap in `defineConfig` or is the plan syntax acceptable? I can't remember tbh